cross/sslh/Makefile

```diff
@@ -8,19 +8,24 @@ PKG_DIST_SITE = https://github.com/yrutschle/sslh/archive
 PKG_DIST_FILE = $(PKG_NAME)-$(PKG_DIST_NAME)
 PKG_DIR = $(PKG_NAME)-$(PKG_VERS)
 
-DEPENDS = cross/libconfig cross/pcre
+DEPENDS = cross/libconfig cross/pcre cross/libcap
 
 HOMEPAGE = http://www.rutschle.net/tech/sslh.shtml
 COMMENT  = sslh accepts HTTPS, SSH and OpenVPN connections on the same port
 LICENSE  = GPLv2
 
+PRE_CONFIGURE_TARGET = sslh_pre_configure
 CONFIGURE_TARGET = nop
```

Comment:
As configure target is skipped, I propose to switch `USELIBCAP` variable as `sslh_configure` target

Reply:
Good call, done.